docs/index.rst

```diff
@@ -80,65 +80,72 @@ The best way to get started with Parcels is to have a look at the Jupyter notebo
+
     jupyter notebook
 
 5. The next time you start a terminal and want to work with Parcels, activate the environment with::
 
-    source $HOME/miniconda2/bin/activate py2_parcels
+    source $HOME/miniconda2/bin/activate py2_parcels  # Linux / macOS
+    activate py2_parcels                              # Windows
+
+
+.. _installing-arbitrary-Git-reference:
+
+Installing a non-released version of Parcels
+============================================
 
-An alternative, more advanced option is to use an existing netCDF installation and
-C compiler, ``git clone`` the `master branch of Parcels
-<https://github.com/OceanParcels/parcels>`_
-and ``pip install`` the dependencies in the `environment.yml <https://raw.githubusercontent.com/OceanParcels/parcels/master/environment.yml>`_ file.
+There might be cases where want to install a version of Parcels that has not been released yet.  (Perhaps, if you want to use a bleeding-edge feature which already is included on Github, but not in the conda-forge package.)
 
-.. _installing-windows:
+Then, just after Step 2. of :ref:`installing-parcels` above, remove the conda-forge package again, and use Pip to install Parcels from Github::
+
+    source $HOME/miniconda2/bin/activate py2_parcels_master  # Linux / macOS
+    activate py2_parcels_master                              # Windows
+
+    pip install git+https://github.com/OceanParcels/parcels.git@master
 
-Installing Parcels on Windows
-=============================
 
-The installation process and usage of Parcels is much the same as under :ref:`installing-linux-macos` above. **However, steps 2 and 5 will be different**, as detailed below:
+.. _installation-dev:
 
-2. Download Parcels' `environment_win.yml <https://raw.githubusercontent.com/OceanParcels/parcels/master/environment_win.yml>`_ file from the main Parcels' directory, start the ``Anaconda Prompt`` from the Windows start menu, navigate to the directory where you saved that file, and then run::
+Installation for developers
+===========================
 
-        activate root
-        conda env create -f environment_win.yml -n py2_parcels
-        activate py2_parcels
-        pip install git+https://github.com/OceanParcels/parcels.git@master
+Parcels depends on a working Python installation, a netCDF installation, a C compiler, and various Python packages.  If you prefer to maintain an own Python installation providing all this, ``git clone`` the `master branch of Parcels <https://github.com/OceanParcels/parcels>`_ and ``pip install`` the dependencies in the `environment.yml <https://raw.githubusercontent.com/OceanParcels/parcels/master/environment.yml>`_ file.
 
-5. The next time you start the ``Anaconda Prompt`` and want to work with Parcels, activate the environment with::
+Then, install Parcels in an `editable way <https://pip.pypa.io/en/stable/reference/pip_install/?highlight=editable#cmdoption-e>` by running::
 
-        activate py2_parcels
+    pip install -e .
```

Comment:
Do we need to say anything about $PYTHONPATH here? That was what @AlexCK has real problems with on Slack